src/server/services/servicesUtils.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I would just call this file `utils.js` to be consistent with other naming patterns.

Reply:
fixed